Dockerfile

```diff
@@ -5,6 +5,7 @@ ADD . /endlessh
 WORKDIR /endlessh
 RUN go mod tidy
 RUN CGO_ENABLED=0 go build -o endlessh .
+RUN touch reportedIps.txt && chmod 666 reportedIps.txt
```

Comment:
No longer needed since it should be in memory. Also `666` as permissions is a bad idea...